python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -82,8 +107,11 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         logger.info("Initializing")
         Ganga.Runtime._prog.initEnvironment()
     else:
-        from Ganga.Runtime.Repository_runtime import startUpRegistries
+        # Load the config from the appropriate files
+        load_config_files()
```

Comment:
Why are we doing something different here to loading the Config in the same way as ```ganga``` and bootstrap?
I'd much rather see us have some consistent way of loading the config between all 3 than 3 special cases of how the config is to be handled.
If there is a solid reason why this is special then OK.